buildエラーの解決

@claude
プルリクエストをマージした際、以下のエラーが発生しました。
原因を特定し解決してください。
```
Run npm run build

> github-pages-demo@0.1.0 build
> next build

[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

   ▲ Next.js 16.0.4 (Turbopack)

   Creating an optimized production build ...
[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
 ✓ Compiled successfully in 2.7s
   Running TypeScript ...
Failed to compile.

./app/household/page.tsx:371:40
Type error: Argument of type 'string' is not assignable to parameter of type 'SetStateAction<"給与" | "賞与" | "副業" | "その他収入" | "食費" | "住居費" | "光熱費" | "交通費" | "通信費" | "娯楽費" | "医療費" | "その他支出">'.

  369 |         <select
  370 |           value={category}
> 371 |           onChange={(e) => setCategory(e.target.value)}
      |                                        ^
  372 |           className="w-full px-3 py-2 rounded-lg border border-slate-300 focus:ring-2 focus:ring-blue-500"
  373 |         >
  374 |           {categories.map(cat => (
Next.js build worker exited with code: 1 and signal: null
Error: Process completed with exit code 1.
```

--- a/app/household/page.tsx
+++ b/app/household/page.tsx
@@ -242,7 +242,7 @@ function OverviewTab({ summary, onAddTransaction }: OverviewTabProps) {
             </label>
             <select
               value={category}
-              onChange={(e) => setCategory(e.target.value)}
+              onChange={(e) => setCategory(e.target.value as IncomeCategory | ExpenseCategory)}
               className="w-full px-4 py-3 rounded-lg border border-slate-300 focus:ring-2 focus:ring-blue-500 focus:border-blue-500"
               required
             >
@@ -368,7 +368,7 @@ function TransactionItem({ transaction, isEditing, onEdit, onCancelEdit, onUpdat
         </div>
         <select
           value={category}
-          onChange={(e) => setCategory(e.target.value)}
+          onChange={(e) => setCategory(e.target.value as IncomeCategory | ExpenseCategory)}
           className="w-full px-3 py-2 rounded-lg border border-slate-300 focus:ring-2 focus:ring-blue-500"
         >
           {categories.map(cat => (

